### ISSUE TITLE
Exception Value: Reverse for 'userena_profile_detail' with arguments...

### ISSUE THREAD

**Issue by marcbelmont** [bread-and-pepper/django-userena#502]
Choosing a username that starts with signup (like signupxyz) will produce an error.

django-userena==1.4.0
Django==1.7.10

```
Traceback:
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  137.                 response = response.render()
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/response.py" in render
  103.             self.content = self.rendered_content
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/response.py" in rendered_content
  80.         content = template.render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/base.py" in render
  148.             return self._render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/base.py" in _render
  142.         return self.nodelist.render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/base.py" in render
  844.                 bit = self.render_node(node, context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/debug.py" in render_node
  80.             return node.render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/loader_tags.py" in render
  126.         return compiled_parent._render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/base.py" in _render
  142.         return self.nodelist.render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/base.py" in render
  844.                 bit = self.render_node(node, context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/debug.py" in render_node
  80.             return node.render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/loader_tags.py" in render
  126.         return compiled_parent._render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/base.py" in _render
  142.         return self.nodelist.render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/base.py" in render
  844.                 bit = self.render_node(node, context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/debug.py" in render_node
  80.             return node.render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/defaulttags.py" in render
  312.                 return nodelist.render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/base.py" in render
  844.                 bit = self.render_node(node, context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/debug.py" in render_node
  80.             return node.render(context)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/defaulttags.py" in render
  458.                         six.reraise(*exc_info)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/template/defaulttags.py" in render
  444.             url = reverse(view_name, args=args, kwargs=kwargs, current_app=context.current_app)
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/core/urlresolvers.py" in reverse
  551.     return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))
File "/home/marc/.virtualenvs/autoconv/local/lib/python2.7/site-packages/django/core/urlresolvers.py" in _reverse_with_prefix
  468.                              (lookup_view_s, args, kwargs, len(patterns), patterns))

Exception Type: NoReverseMatch at /accounts/signupxyz/signup/complete/
Exception Value: Reverse for 'userena_profile_detail' with arguments '(<SimpleLazyObject: <User: signupxyz>>,)' and keyword arguments '{}' not found. 1 pattern(s) tried: ['accounts/(?P<username>(?!signout|signup|signin)[\\@\\.\\w-]+)/$']
```

**Comment by swistakm** [comment]
fix released as 1.5.1

**Comment by marcbelmont** [comment]
Thanks!
